src/Components/Dashboard.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react';
+import {useNavigate} from 'react-router-dom';
+import './Dashboard.css';
+function Dashboard()
+{
+    const [query,setQuery]=useState('');
+    const [number,setNumber]=useState(0);
+    const [intolerance,setIntolerance]=useState([]);
+    let navigate=useNavigate();
+    function clickHandler()
+    {
+        localStorage.setItem('query',query);
+        localStorage.setItem('number',number);
+        localStorage.setItem('intolerance',intolerance);
+    }
+    function clickHandler2()

```

Comment:
As i have to call local storage again and again and similarly after creating a method i have to call it similar number of times ...so is this will be optimization of code?